docs/includes/openapi.adoc

```diff
@@ -18,6 +18,155 @@
 
 ifndef::rootdir[:rootdir: {docdir}/..]
 
+// tag::overview[]
+The link:{openapi-spec-url}[OpenAPI specification] defines a standard way to express the interface exposed by a REST service.
+
+The link:{microprofile-open-api-spec-url}[MicroProfile OpenAPI spec] explains how MicroProfile embraces OpenAPI, adding annotations, configuration, and a service provider interface (SPI).
+
+ifdef::mp-flavor[Helidon {flavor-uc} implements the MicroProfile OpenAPI specification.]
+ifdef::se-flavor[OpenAPI support in Helidon {flavor-uc} draws its inspiration from MicroProfile OpenAPI but does not implement the spec because Helidon {flavor-uc} does not support annotations.]
+
+The OpenAPI support in Helidon {flavor-uc} performs two main tasks:
+
+* Build an in-memory model of the REST API your service implements.
+* Expose the model in text format (typically yaml) via the `/openapi` endpoint.
+
+To construct the model, Helidon OpenAPI gathers information about the service API from whichever of these sources are present in the application:
```

Comment:
Just "Helidon" instead of "Helidon OpenAPI", or the "the Helidon OpenAPI implementation"?

Reply:
fixed